commands/pack.js

```diff
@@ -147,8 +147,8 @@ function copyPublicFolder(output) {
 // Print a detailed summary of build files.
 function printFileSizes(stats, output) {
 	const assets = stats
-		.toJson({all: false, assets: true})
-		.assets.filter(asset => /\.(js|css|bin)$/.test(asset.name))
+		.toJson({assets: true})
+		.assets.filter(asset => /^(?!.*chunk).*\.(js|css|bin)$/.test(asset.name))
```

Comment:
Is this chunk file in the dist folder?
How can I reproduce?

Reply:
Ah...I see. Let's not add this at this moment.

Reply:
OK